### PR TITLE
Remove all uses of ask_for_values

### PR DESCRIPTION
**Pull request by rassouly**
`ask_for_values` has been deprecated by the pyvisa library so this patch updates all uses of `ask` to `query`.

Currently tested:
  - Anapico
  - Anritsu
  - TinyBilt
  - R&S SMB100A
  - Agilent PSG
  - PNA
  - WindfreakTech SynthHD

### CODE REVIEW

**Comment by rassouly** [comment]
I've tested this patch for most of the instruments I have access to (with the exceptions of the AWG since the current drivers are not on master), and it seems to work

**Comment by MatthieuDartiailh** [comment]
@rassouly can you list which driver you did not test ? @galactikvoyager could somebody on your side check the remaining drivers ? I am not comfortable merging without some sanity check on your side first.

**Comment by rassouly** [comment]
I did not test any multimeter (Agilent or Keysight), the Cryomagnetics and Oxford current sources, the R&S VNA and ZVA and the SR lock-in. 
Some of the drivers in the folder seem to not be registered in the manifest such as the LeCroy oscilloscope (we have one but we use it manually), the agilent PSA (we've just bought one, the driver looks a bit weird but I could try it).
As I said previously, I didn't bother testing the AWGs since the drivers on this branch are quite outdated.

**Comment by MatthieuDartiailh** [comment]
Thanks @rassouly, I guess @leghtas should have a look then for the R&S instruments.

**Comment by rassouly** [comment]
The new version of Pyvisa has already dropped support for `ask_for_values`and nobody in Paris has commented in 2 months. Currently a new installation of exopy is not functional because of this change.
Can you merge this now? If Paris has issues, they can fix it later.

File: exopy_hqc_legacy/instruments/drivers/visa_tools.py
Comment (MatthieuDartiailh) [inline exopy_hqc_legacy/instruments/drivers/visa_tools.py]: This should simply be removed since it will crash.

File: exopy_hqc_legacy/instruments/drivers/visa_tools.py
Comment (MatthieuDartiailh) [inline exopy_hqc_legacy/instruments/drivers/visa_tools.py]: I suggest also getting rid of this.

**Review by MatthieuDartiailh — CHANGES_REQUESTED** [review]
I suggested two minor changes I will merge once you address them.

**Comment by rassouly** [comment]
The build error is simply a timeout from the conda website when travis was trying to install exopy from conda.

**Review by MatthieuDartiailh — APPROVED** [review]

**Comment by MatthieuDartiailh** [comment]
BTW can you add a releasenote about this change as part of #72 ?

**Pull request merged**
